public/content/en/vibed/deploy-on-heroku.md

```diff
@@ -0,0 +1,183 @@
+and depending on the programming language, the scripts will retrieve dependencies, 
+output generated assets or compiled code, and more.
+
+[More information](https://devcenter.heroku.com/articles/buildpacks)
+
+We are going to use this [webpack](https://github.com/skirino/heroku-buildpack-vibe.d) it uses dmd by default. 
+
+```
+$ heroku buildpacks:set https://github.com/skirino/heroku-buildpack-vibe.d.git#cedar-14
+```
+
+You should also create a file called `vibed_buildpack.config` in the root of your app. 
+We are doing this because this buildpack's dependencies are outdated. 
+
+Fortunately we can override them by providing new urls : 
+
+```
+DMD_ARCHIVE_URL = http://downloads.dlang.org/releases/2016/dmd.2.070.2.zip
+DUB_ARCHIVE_URL = http://code.dlang.org/files/dub-0.9.25-linux-x86_64.tar.gz
+```
+
+### Deploy the code 
+
+```
+$ git add .
+$ git commit -am "make it better"
+$ git push heroku master
+Counting objects: 9, done.
+Delta compression using up to 2 threads.
+Compressing objects: 100% (6/6), done.
+Writing objects: 100% (9/9), 997 bytes, done.
+Total 9 (delta 0), reused 0 (delta 0)
+
+-----> Fetching custom git buildpack... done
+-----> D (dub package manager) app detected
+-----> Building libevent
+-----> Building libev
+-----> Downloading DMD
+-----> Downloading dub package manager
+-----> Setting PATH:
+-----> Initializing toolchain
+-----> Building app
+       Running dub build ...
+Building configuration "application", build type release
+Running dmd (compile)...
+Compiling diet template 'index.dt' (compat)...
+Linking...
+       Build was successful
+-----> Discovering process types
+       Procfile declares types -> web
+-----> Compiled slug size: 3.5MB
+-----> Launching... done, v4
+       https://rocky-hamlet-67506.herokuapp.com/ deployed to Heroku
+To git@heroku.com:rocky-hamlet-67506.git
+ * [new branch]      master -> master
+```
+
+All you need to do now is open the app in the browser
+
+```
+$ heroku open
+```
+
+### Scale the app 
+
+Right now, our app is running on a single web dyno. Think of a dyno as a lightweight container that runs the command specified in the Procfile.
+You can check how many dynos are running using the ps command:
+
+```
+$ heroku ps
+Free dyno hours quota remaining this month: 550h 0m (100%)
+For more information on dyno sleeping and how to upgrade, see:
+https://devcenter.heroku.com/articles/dyno-sleeping
+
+No dynos on ⬢ rocky-hamlet-67506
+```
+
+By default, your app is deployed on a free dyno. 
+Free dynos will sleep after a half hour of inactivity (if they don’t receive any traffic). This causes a delay of a few seconds for the first request upon waking. 
+
+To make it work you should scale it up using 
+
+```
+$ heroku ps:scale web=1
+```
+ 
+### See the logs 
+
+Heroku treats logs as streams of time-ordered events aggregated from the output streams of all your app and Heroku components, 
+providing a single channel for all of the events.
+
+```
+$ heroku logs --tail
+```
```

Comment:
maybe add some links how to proceed from here, e.g.
- adding addon (postgres, mongo, logging)
- more heroku documentation links (e.g. use your own domain)